argilla-frontend/v1/domain/entities/record/Record.ts

```diff
@@ -20,7 +20,10 @@ export class Record {
     public answer: RecordAnswer,
     private readonly suggestions: Suggestion[],
     score: number,
-    public readonly page: number
+    public readonly page: number,
+    public readonly metadataInsertedAt: string,
+    public readonly metadataUpdatedAt: string,
+    public readonly metadata: Object
```

Comment:
so, why the metadata is an "anonymous" object and why if you pass the metadata as an object you passed the metadataInsertedAt and metadataUpdatedAt in other params?